src/mlpack/methods/kmeans/naive_kmeans_impl.hpp

```diff
@@ -56,8 +55,8 @@ double NaiveKMeans<DistanceType, MatType>::Iterate(const arma::mat& centroids,
 
       for (size_t j = 0; j < centroids.n_cols; ++j)
       {
-        const double dist = distance.Evaluate(dataset.col(i),
-            centroids.unsafe_col(j));
+        const double dist = distance.Evaluate(dataset.col(i), 
```

Comment:
```suggestion
        const double dist = distance.Evaluate(dataset.col(i),
```
This is super pedantic :smile: